styles/products.css

```diff
@@ -0,0 +1,64 @@
+.filters {
+  display: flex;
+  position: fixed;
+  flex-direction: column;
+  width: 20%;
+  padding: 1rem;
+  height: 100vh;
+  z-index: 10;
```

Comment:
A better way to implement the sidebar filters could be with grid? Maybe using 20-80 split (for desktop)
Else not really sure of the purpose of using zindex here

Reply:
Added z-index because i have set the position as fixed and sometimes another components overlaps it and the sidebar is not visible. Will try to implement it using grid later...seems efficient!